src/System.Drawing.Common/tests/FontTests.cs

```diff
@@ -16,6 +16,42 @@ public static IEnumerable<object[]> Ctor_Family_Size_TestData()
             yield return new object[] { FontFamily.GenericSerif, float.MaxValue };
         }
 
+        public static IEnumerable<object[]> FontFamily_Equals_SameFamily_TestData()
+        {
+            yield return new object[] { FontFamily.GenericMonospace, 1 };
+            yield return new object[] { FontFamily.GenericSerif, float.MaxValue };
+            yield return new object[] { FontFamily.GenericSansSerif, 10 };
+        }
+
+        public static IEnumerable<object[]> FontFamily_Equals_DifferentFamily_TestData()
+        {
+            yield return new object[] { FontFamily.GenericMonospace, FontFamily.GenericSerif };
+            yield return new object[] { FontFamily.GenericSansSerif, FontFamily.GenericSerif };
+            yield return new object[] { FontFamily.GenericSansSerif, FontFamily.GenericMonospace };
+        }
+
+        [ConditionalTheory(Helpers.GdiplusIsAvailable)]
+        [MemberData(nameof(FontFamily_Equals_SameFamily_TestData))]
+        public void Font_Equals_SameFontFamily(FontFamily fontFamily, float size)
+        {
+            using (var font1 = new Font(fontFamily, size))
+            using (var font2 = new Font(fontFamily, size))
+            {
+                Assert.True(font1.Equals(font2));
+            }
+        }
+
+        [ConditionalTheory(Helpers.GdiplusIsAvailable)]
+        [MemberData(nameof(FontFamily_Equals_DifferentFamily_TestData))]
+        public void Font_Equals_DifferentFontFamily(FontFamily fontFamily1, FontFamily fontFamily2)
+        {
+            using (var font1 = new Font(fontFamily1, 9))
+            using (var font2 = new Font(fontFamily2, 9))
+            {
+                Assert.False(font1.Equals(font2));
```

Comment:
I see on 3 distros this is failing - for 3 different font families. 

I guess on these in some cases 2 of these give the same result. 
```
                    status = SafeNativeMethods.Gdip.GdipGetGenericFontFamilySerif(out nativeFamily);
                    status = SafeNativeMethods.Gdip.GdipGetGenericFontFamilySansSerif(out nativeFamily);
                    status = SafeNativeMethods.Gdip.GdipGetGenericFontFamilyMonospace(out nativeFamily);
```

Not worth bother too much today. You could just check that Equals never returns true if GetHashCode returns different values. That's something.

Reply:
I see what is happening, I have a docker on Fedora and just finished looking. It seems like the GenericSansSerif and GenericSerif on those distros are the same `FontFamily`

Reply:
Actually all the Generic fonts return the same values on Fedora.